### PR TITLE
feat: list all raids on /raids

### PR DESCRIPTION
**Pull request by nobrayner**
As title + fixing up of formatting and things

### CODE REVIEW

File: frontend/src/App.svelte [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
-	<Route path="raids" component={Raids} />
-	<Route path="raids/:id" component={ViewRaid} />
-	<Route path="*" component={NotFound} />
+  <Route path="/">
+    <h1>Welcome to the Open Source Raid Guild</h1>
+  </Route>
+  <Route path="quests" component={QuestBoard} />
+  <Route path="raids" component={Raids} />
+  <Route path="raids/:id" component={ViewRaid} />
+  <Route path="*" component={NotFound} />
```
Comment (nobrayner) [inline frontend/src/App.svelte]: Ah. Tab spacing changed or something. VSCode didn't show any diffs :sweat_smile:

Comment (codyarose) [reply frontend/src/App.svelte]: Yeah, I figured more specific Prettier rules might be necessary 😂

Comment (JacobMGEvans) [reply frontend/src/App.svelte]: Oh remove the ESLint rule for tabs and just let Prettier handle it

**Comment by nobrayner** [comment]
Super basic (read: ugly) listing of all raids on /raids

I think this should be merged in now. I will finish off the wireframe for this screen, and get someone on it.

**Review by JacobMGEvans — APPROVED** [review]
Remove tabs

**Pull request merged**